APMrover2/APMrover2.cpp

```diff
@@ -392,10 +391,6 @@ void Rover::update_GPS_10Hz(void)
         set_system_time_from_GPS();
 
         if (gps.status() >= AP_GPS::GPS_OK_FIX_3D) {
```

Comment:
sounds good to remove. it's an existing issue of course, not related to this patch (except that this patch happens to touch lines nearby).